test/gemm/gemm_coop_schedule.hpp

```diff
@@ -50,11 +50,11 @@ namespace rocwmma
 
                 constexpr static inline auto waveIndex()
                 {
-                    return std::get<1>(WaveSpace::localWaveCoord());
+                    return get<1>(WaveSpace::localWaveCoord());
                 }
-                constexpr static inline auto waveCount()
+                constexpr static inline uint32_t waveCount()
```

Comment:
uint32_t specification required? E.g. waveIndex does not have this. Just curious for consistency